apps/api/schedule.py

```diff
@@ -17,10 +17,16 @@ def _require_video_api_key(func):
     @wraps(func)
     def wrapper(*args, **kwargs):
         auth_header = request.headers.get("authorization", None)
-        if not auth_header or not auth_header.startswith("Bearer "):
+        if not auth_header:
+            abort(401)
+
+        if auth_header.startswith("bearer "):
+            bearer_token = auth_header.removeprefix("bearer ")
```

Comment:
The contents of the Authorization header is case-sensitive, so "bearer" should be wrong. Can you add a comment to explain why this is needed?

Reply:
I added it because I wasted 5 minutes scratching my head why my curl request was giving a 401 ;-)

I went looking for consensus online, always a mistake.. honestly, it seems like a bit of a mess: https://sgeb.io/posts/fix-go-oauth2-case-sensitive-bearer-auth-headers/

Given we're not implementing oauth here I figured it was a convenience and didn't worry about it too much.

Reply:
Ahh, fair, `# make it easier with curl` would be fine then